packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -15,12 +15,29 @@ import 'scroll_simulation.dart';
 
 export 'package:flutter/physics.dart' show Tolerance;
 
+/// Determines the physics of a [Scrollable] widget.
+///
+/// For example, determines how the [Scrollable] will behave when the user
+/// reaches the maximum scroll extent or when the user stops scrolling.
```

Comment:
The docs for particle/position/velocity talk about a "particle". We should probably explain the particle angle here or define those methods in terms of the simulation's position; and then mention that position is usually mapped to scroll offset.